examples/04_pong/main.rs

```diff
@@ -202,18 +210,21 @@ impl<'a> System<'a> for PongSystem {
                 println!("Left player score: {0}, Right player score {1}",
                          score.score_left,
                          score.score_right);
+                play::once(&self.score_sfx, &output::default_output().unwrap()).unwrap();
             }
 
             // Check if the ball is below the top boundary, if it is not deflect it
             if ball.position[1] + ball.size / 2. > top_bound {
                 ball.position[1] = top_bound - ball.size / 2.;
                 ball.velocity[1] = -ball.velocity[1];
+                play::once(&self.bounce_sfx, &output::default_output().unwrap()).unwrap();
             }
 
             // Check if the ball is above the bottom boundary, if it is not deflect it
             if ball.position[1] - ball.size / 2. < bottom_bound {
                 ball.position[1] = bottom_bound + ball.size / 2.;
                 ball.velocity[1] = -ball.velocity[1];
+                play::once(&self.bounce_sfx, &output::default_output().unwrap()).unwrap();
```

Comment:
Yeah I was just working in the existing framework of the pong example :P I'm happy to refactor this to use a physics system as soon as we have it.